crates/mun_hir/src/ty/tests.rs

```diff
@@ -51,6 +51,18 @@ fn void_return() {
     )
 }
 
+#[test]
+fn place_expressions() {
+    infer_snapshot(
+        r#"
+    fn foo(a:int) {
+        a += 3;
```

Comment:
To be "feature-complete" it would be good to also add `-=`, `*=`, and `/=` to the test.

Reply:
Ill add another test specifically for that